packit_service/worker/jobs.py

```diff
@@ -343,12 +202,12 @@ def report_task_accepted(
         elif isinstance(handler, ProposeDownstreamHandler):
             job_helper = ProposeDownstreamJobHelper(
                 service_config=self.service_config,
-                package_config=event.package_config,
-                project=event.project,
-                metadata=EventData.from_event_dict(event.get_dict()),
-                db_trigger=event.db_trigger,
+                package_config=self.event.package_config,
+                project=self.event.project,
+                metadata=EventData.from_event_dict(self.event.get_dict()),
+                db_trigger=self.event.db_trigger,
                 job_config=job_config,
-                branches_override=event.branches_override,
+                branches_override=self.event.branches_override,
             )
             task_accepted_time = datetime.now(timezone.utc)
```

Comment:
I would move this `task_accepted_time` before if-statement since it's present in if and elif block so we can use it there only once.